tests/org.jboss.tools.maven.ui.bot.test/src/org/jboss/tools/maven/ui/bot/test/utils/ProjectHasNature.java

```diff
@@ -45,6 +46,12 @@ public boolean test() {
 		pd.open();
 		new WaitUntil(new ShellIsAvailable("Properties for " + projectName), TimePeriod.DEFAULT);
 		new DefaultTreeItem("Project Facets").select();
+		try {
+			new WaitUntil(new ShellIsAvailable("Setting Java Build Path"), TimePeriod.DEFAULT);
```

Comment:
I suppose this will not be needed when https://github.com/jbosstools/jbosstools-integration-tests/pull/1944/files#r280289197 is resolved.

Reply:
Rewritten as we discussed.